plugins/package-managers/maven/src/main/kotlin/tycho/P2RepositoryContentLoader.kt

```diff
@@ -135,14 +135,14 @@ internal class P2RepositoryContentLoader : AutoCloseable {
          */
         private fun parseArtifactsFile(file: File): Map<String, Hash> {
             val handler = ElementHandler(ParseArtifactsState())
-                .handleElement("artifact") { state, attributes ->
+                .handleElement("artifact") { state, attributes, _ ->
```

Comment:
Commit message nit:

>  be slightly extended because it is now necessary to deal with

Superfluous space before "be".

Reply:
Did you forget to push? The space still seems to be there.

Reply:
Sorry, I was sure I had fixed it. Now it should be gone.